pkg/types/hashedrekord/v0.0.1/hashedrekord_v0_0_1_schema.json

```diff
@@ -15,11 +15,11 @@
                     "format": "byte"
                 },
                 "publicKey" : {
-                    "description": "The public key that can verify the signature",
+                    "description": "The public key that can verify the signature; this can also be an X509 code signing certificate that contains the raw public key information",
```

Comment:
nit: specify encoding?
base64 encoding of the PEM encoded public key or X.509 cert right?

Reply:
the `format: byte` below says that they should be base64 encoded. I think PEM or DER will work for the raw key, and probably the same for the cert.

Reply:
Ah you're right, DER works as well, LGTM then